okhttp/src/jvmMain/kotlin/okhttp3/internal/connection/RouteException.kt

```diff
@@ -1,33 +0,0 @@
-/*
```

Comment:
There are a small number of references of this type in github, I know it's internal but is it worth deprecating instead of removing.  I'll leave call to you.  But you could make the argument that the exception types thrown from a public API are public API also.

Reply:
I believe that in all recent releases, this is thrown and caught always within OkHttp. End users who are catching it should be able to stop cause that is now dead code.

I don't like breaking binary compatibility, but this is in internal and we can't be paralyzed there.

Reply:
Can you share your search results? I am bad at GitHub search! Everything I found was just decompiles of OkHttp itself.

Reply:
Let's do it then.

It's mostly as you describe https://github.com/search?p=1&q=%22import+okhttp3.internal.connection.RouteException%22&type=Code

https://github.com/NBXXF/xxf_android/blob/777162e3412bcf75ab1f41361d6fce240cb1952f/xxf/src/main/java/com/xxf/arch/rxjava/transformer/filter/ErrorIgnoreNetFilter.kt
https://github.com/nilam24/SmartPantry/blob/19368abfacd8288c155656c3eda053f9e36d772a/app/src/main/java/com/example/src/smartpantry/ApiClient.java

Nothing big, like the Google Maps SDK or similar.